packages/compass-components/src/components/spin-loader.tsx

```diff
@@ -24,22 +23,16 @@ const spinLoaderStyle = css`
   animation: ${shellLoaderSpin} 700ms ease infinite;
 `;
 
-export default class SpinLoader extends Component<SpinLoaderProps> {
-  static defaultProps = {
-    size: '12px',
-  };
-
-  render(): JSX.Element {
-    const { size } = this.props;
-
-    return (
-      <div
-        className={spinLoaderStyle}
-        style={{
-          width: size,
-          height: size,
-        }}
-      />
-    );
-  }
+function SpinLoader({ size = '12px' }: SpinLoaderProps): JSX.Element {
```

Comment:
I know this is not part of the PR but shouldn't this be `spacing[something]`?

Reply:
It would be `spacing[2] * 1.5` so I figured we can update it when we're looking at that component next if that's alright instead of adding in extra logic here.